deploy/osd-26960/00-roles.yaml

```
@@ -0,0 +1,75 @@
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: sre-operator-reinstall-sa
+  namespace: openshift-route-monitor-operator
```

This needs to be updated to OAO
```suggestion
  namespace: openshift-ocm-agent-operator
```